ci-operator/config/openshift/cincinnati-graph-data/openshift-cincinnati-graph-data-master.yaml

```diff
@@ -53,6 +54,7 @@ tests:
     timeout: 1h0m0s
     version: "4.14"
   optional: true
+  run_if_changed: internal-channels/candidate.yaml
```

Comment:
@wking This is the one I was least certain about running only on candidate changes, does this scan the entire registry? I'd really like to avoid scanning all quay.io tags in these tests whenever possible provided we're not creating a giant blind spot

Reply:
`e2e-latest-cincinnati` does scan the registry, while it checks to see if modern Cincinnati [can still serve a valid graph from the proposed graph-data][1].  It's `optional: true`, so maybe we should move it to a periodic or something?  I'm not clear on what sorts of graph-data changes we'd make that might break this.  But because it's optional and I don't really know what it's looking for, I'm fine with `run_if_changed: ${ANYTHING}` to save Quay load by running less often.  #26639 added the presubmit, but doesn't describe why.  CC @PratikMahajan @DavidHurta, who have a better grasp than me on what `e2e-latest-cincinnati` is trying to protect us from.

[1]: https://github.com/openshift/cincinnati/blob/074fbe1271a741d61ea15064dd0532a7354284c4/hack/e2e.sh#L3-L6

Reply:
> I'm not clear on what sorts of graph-data changes we'd make that might break this. But because it's optional and I don't really know what it's looking for, I'm fine with run_if_changed: ${ANYTHING} to save Quay load by running less often.

I agree. However, I am quite new to the Cincinnati project, so take my opinion with a grain of salt.